petpal/comments/views.py

```diff
@@ -60,7 +73,13 @@ def comment_create_view(request):
             rating = request.data.get('rating')
             new_comment = Comment(content=content, is_author_seeker=is_author_seeker, seeker=seeker, shelter=shelter, is_review=is_review, rating=rating)
             new_comment.save()
-            return Response({'msg': 'Review Created'}, status=status.HTTP_200_OK)
+            data = {
```

Comment:
Looks good, I don't think I took into account that a shelter can respond to a review when I made the initial implementation, I think it should work when review is false and no rating but just want to make sure this case is handled

Reply:
If there's no rating then rating will just be null it won't throw an error.